Specification.md

```diff
@@ -55,21 +66,30 @@ A *should-total* MUST be a *duration* value
 followed by a `!`
 and wrapped in “parentheses”.
 
-### Summary
-A *summary* is user-provided text for holding arbitrary information.
-
-There are two places where *summary* text MAY appear in *records*:
-- Underneath the *date*:
-  In this case the *summary* is considered to be associated with the entire *record*.
-  The *summary* MAY span multiple lines.
-  Each of its lines MUST NOT start with “blank characters”.
-- Behind *entries*:
-  In this case the *summary* is only considered to be referring to the corresponding *entry*.
-  The *summary* text follows the *entry* on the same line,
-  and it ends at the end of that line.
-  It MUST be separated from the *entry* by one “space”.
-
-### Tags
+### Summaries
```

Comment:
Although this third level heading is referring to two types of summaries, I still think it is a odd that unlike every other heading, it is being used in plural, while the others use singular.

Reply:
Ah, good spot. There was no reasoning behind that, I’ve converted that headline (and 2 others) to be singular.